samples/perf_client/perf_client.h

```diff
@@ -452,8 +452,8 @@ namespace client
       cli::add_address_option(
         app,
         server_address,
-        "--server-address",
-        "Remote node RPC server address where requests should be sent")
+        "--rpc-address",
```

Comment:
I have updated this here and throughout our clients so that it is consistent with the `--rpc-address` that is passed by `cchost` to specify the RPC listening address.